_includes/about.html

```diff
@@ -0,0 +1,18 @@
+<head>
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.5.0/css/all.css" integrity="sha384-B4dIYHKNBt8Bc12p+WXckhzcICo0wtJAoU8YZTY5qE0Id1GSseTk6S+L3BlXeVIU" crossorigin="anonymous">
+</head>
+<div class="col s12 tag-div">
+  <div class="user-pic center-align">
+    <a href="{{site.github_profile}}" target="_blank"><img class="circle hoverable z-depth-1" src="{{site.baseurl}}/assets/res/user-red.png"></a>
+  </div>
+  <div class="user-content">
+    <div class="about-text">
+      <p><i>Find more about us</i></p>
+      <span>
+        <a href="https://github.com/HotelsDotCom" class="github-icon" target="_blank"><i class="fab fa-github"></i></a>
+        <a href="https://twitter.com/HcomTechnology" class="twitter-icon" target="_blank"><i class="fab fa-twitter"></i></a>
+        <a href="https://medium.com/hotels-com-technology" class="medium-icon" target="_blank"><i class="fab fa-medium"></i></a>
+      </span>
+    </div>
+  </div>
+</div>
```

Comment:
Minor, but it's good practice to ensure all files end with a line break (https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)

Reply:
You missed this one!

Reply:
True, added a new line!